tests/unittests/Foundation/NSDecimalNumberTests.mm

```diff
@@ -0,0 +1,332 @@
+//******************************************************************************
```

Comment:
fair, but at the end the NSDecimal needs to be created, which is the exact tests that go through NSDecimal*

---
In reply to: [80601818](https://github.com/Microsoft/WinObjC/pull/1032#discussion_r80601818) [](ancestors = 80601818)